plugins/Toolbox/resources/qml/ToolboxInstalledPage.qml

```diff
@@ -1,50 +1,40 @@
-// Copyright (c) 2018 Ultimaker B.V.
+// Copyright (c) 2019 Ultimaker B.V.
 // Toolbox is released under the terms of the LGPLv3 or higher.
 
 import QtQuick 2.10
-import QtQuick.Dialogs 1.1
-import QtQuick.Window 2.2
-import QtQuick.Controls 1.4
-import QtQuick.Controls.Styles 1.4
+import QtQuick.Controls 2.3
 
 import UM 1.1 as UM
 
 ScrollView
 {
     id: page
-    frameVisible: false
+    clip: true
     width: parent.width
     height: parent.height
-    style: UM.Theme.styles.scrollview
-    flickableItem.flickableDirection: Flickable.VerticalFlick
 
     Column
     {
+        width: page.width
         spacing: UM.Theme.getSize("default_margin").height
+        padding: UM.Theme.getSize("wide_margin").width
         visible: toolbox.pluginsInstalledModel.items.length > 0
-        height: childrenRect.height + 4 * UM.Theme.getSize("default_margin").height
-
-        anchors
-        {
-            right: parent.right
-            left: parent.left
-            margins: UM.Theme.getSize("default_margin").width
-            top: parent.top
-        }
+        height: childrenRect.height + 2 * UM.Theme.getSize("wide_margin").height
 
         Label
         {
-            width: page.width
+            width: parent.width - 2 * parent.padding
```

Comment:
It's better to anchor it (according to the QML best practices, it's faster that way).
As it's parent's width is already set, it would be something like
anchors.left: parent.left
anchors.right: parent.right
anchors.margins: parent.padding